bundle/manifests/tempo.grafana.com_microservices.yaml

```diff
@@ -580,6 +580,75 @@ spec:
           status:
             description: MicroservicesStatus defines the observed state of Microservices.
             properties:
+              conditions:
+                description: Conditions of the Tempo deployment health.
+                items:
+                  description: "Condition contains details for one aspect of the current
+                    state of this API Resource. --- This struct is intended for direct
```

Comment:
Is there a way to ignore this description? This is coming from the comment in `[]metav1.Condition`.

Reply:
It seems you're not the first one finding this "issue": https://docs.openshift.com/container-platform/4.11/rest_api/operatorhub_apis/operatorcondition-operators-coreos-com-v2.html#spec